requests/sessions.py

```diff
@@ -10,7 +10,11 @@
 """
 import os
 from collections import Mapping
-from datetime import datetime
+from datetime import timedelta
+try:
+    from time import monotonic
+except:
```

Comment:
This code should be moved into https://github.com/kennethreitz/requests/blob/master/requests/compat.py